### PR TITLE
AdminPanel : Custom warning message

### PR DESCRIPTION
**Pull request by generalgui**
Convert static Admin panel Warning into a custom Warning message

### CODE REVIEW

**Comment by generalgui** [comment]
http://51.178.133.139:8080/projects/alliance-v0-dot-4/work_packages/71/activity

**Comment by generalgui** [comment]
op#71

**Comment by generalgui** [comment]
OP#71

File: Alliance.Client/Extensions/AdminMenu/ViewModels/AdminVM.cs [hunk 1]
```diff
@@ -1,491 +1,513 @@
...
+
+		public void SendWarningToPlayer(string customWarning)
+		{
+			if (_selectedPeer == null) { return; }
+			GameNetwork.BeginModuleEventAsClient();
+			GameNetwork.WriteMessage(new AdminClient() { SendWarningToPlayer = true, PlayerSelected = _selectedPeer.PeerId, WarningMessageToPlayer = customWarning });
+			GameNetwork.EndModuleEventAsClient();
+		}
+
+		public void CustomWarning()
```
Comment (pierreLemp) [inline Alliance.Client/Extensions/AdminMenu/ViewModels/AdminVM.cs:338]: I would rename this method to something like `PrompWarningMessageSelection`

File: Alliance.Server/Extensions/AdminMenu/Handlers/AdminMenuHandler.cs [hunk 2]
```diff
@@ -270,11 +270,11 @@ public bool SendWarningToPlayer(NetworkCommunicator peer, AdminClient admin)
 			if (playerSelected == null) return false;
 
 			GameNetwork.BeginModuleEventAsServer(playerSelected);
-			GameNetwork.WriteMessage(new SendNotification($"Vous avez reçu un avertissement d'un Admin ({peer.UserName}) !", 0));
+			GameNetwork.WriteMessage(new SendNotification($"{admin.WarningMessageToPlayer} (Admin : {peer.UserName}) !", 0));
```
Comment (pierreLemp) [inline Alliance.Server/Extensions/AdminMenu/Handlers/AdminMenuHandler.cs:273]: Maybe you should handle the case when there is no message inside `admin.WarningMessageToPlayer`.
In order to have a default message.

**Review by pierreLemp — COMMENTED** [review]
Nice job !

File: Alliance.Client/Extensions/AdminMenu/ViewModels/AdminVM.cs (hunk 1 above)
Comment (generalgui) [reply Alliance.Client/Extensions/AdminMenu/ViewModels/AdminVM.cs]: Done into AdminVM.cs & AdminPanel.xml

File: Alliance.Server/Extensions/AdminMenu/Handlers/AdminMenuHandler.cs (hunk 2 above)
Comment (generalgui) [reply Alliance.Server/Extensions/AdminMenu/Handlers/AdminMenuHandler.cs:275]: Done : 
Added : admin.WarningMessageToPlayer = string.IsNullOrEmpty(admin.WarningMessageToPlayer) ? "Vous avez reçu un avertissement d'un Admin" : admin.WarningMessageToPlayer;

**Review by pierreLemp — APPROVED** [review]

**Review by Pierre-HenriLemp — APPROVED** [review]

**Pull request merged**